include/boost/geometry/algorithms/discrete_frechet_distance.hpp

```diff
@@ -225,15 +225,15 @@ struct discrete_frechet_distance<default_strategy, false>
 
 
 /*!
-\brief Calculate discrete Frechet distance between two geometries (currently
-       works for LineString-LineString) using specified strategy.
+\brief \brief_calc2{discrete Frechet distance, between} \brief_strategy
+\details \details_free_function{discrete_frechet_distance, discrete Frechet distance, between}.
```

Comment:
This was written in heading `Behavior` but for all other functions it is different. Harmonized that and the parameters below.
Also the support is now as usual (not automatic though)